db/migrate/20161129222053_polymorphize_quotes.rb

```diff
@@ -0,0 +1,8 @@
+class PolymorphizeQuotes < ActiveRecord::Migration
+  def change
+    rename_column :quotes, :anime_id, :media_id
+    add_column :quotes, :media_type, :string, null: false
```

Comment:
When adding a polymorphism, you kinda gotta do it in three steps:

1. add `media_type`
2. fill `media_type` with `Anime`
3. set `null: false`

I think `change_column_null` can handle both 2 and 3 if you use the `default` param